### PR TITLE
Improve release publicity

### PR DESCRIPTION
**Pull request by drewbo**
## Changes proposed in this pull request:
- Adds a Github release on production deploys
- Minor documentation updates

## security considerations
None

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+## temp
+
+a bunch of nonsense
```
Comment (apburnes) [inline CHANGELOG.md:3]: What is the purpose of this?

**Review by apburnes — COMMENTED** [review]
Quick question about the CHANGELOG.md

File: CHANGELOG.md (hunk 1 above)
Comment (drewbo) [reply CHANGELOG.md:3]: ah this was me testing to make sure incremental release notes could be generated from the changelog. I'll remove

**Review by apburnes — APPROVED** [review]
🚀

**Pull request merged**